lib/package.gi

```diff
@@ -2186,9 +2186,16 @@ InstallGlobalFunction( ValidatePackageInfo, function( info )
         x -> IsString(x) and 0 < Length(x) and x[1] <> '=',
         "a nonempty string that does not start with `='" );
     TestMandat( record, "Date",
-        x -> IsString(x) and Length(x) = 10 and x{ [3,6] } = "//"
-                 and ForAll( x{ [1,2,4,5,7,8,9,10] }, IsDigitChar ),
-        "a string of the form `dd/mm/yyyy'" );
+        x -> IsString(x) and Length(x) = 10
+          and ( ( x{ [3,6] } = "//" and ForAll( x{ [1,2,4,5,7,8,9,10] }, IsDigitChar ) )
+          or ( x{ [5,8] } = "--" and ForAll( x{ [1,2,3,4,6,7,9,10] }, IsDigitChar ) ) ) ,
+        "a string of the form `yyyy-mm-dd` or `dd/mm/yyyy`." );
```

Comment:
Here and in the next message do not use `.` at the end to keep consistent style.